net-analyzer/s3scanner/s3scanner-20190928.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
+
+inherit python-single-r1
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/sa7mon/S3Scanner.git"
+	KEYWORDS=""
+else
+	KEYWORDS="~amd64 ~x86"
+	EGIT_COMMIT="28f6ab4e04ddb55b6e0d2f517ced181d5111072a"
+	SRC_URI="https://github.com/sa7mon/S3Scanner/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+fi
+
+DESCRIPTION="Scan for open AWS S3 buckets and dump the contents"
+HOMEPAGE="https://github.com/sa7mon/S3Scanner"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+MY_PN="S3Scanner"
+
+RDEPEND="dev-python/awscli
+    dev-python/boto3
+    dev-python/coloredlogs
+    dev-python/pytest-xdist
+    dev-python/requests"
```

Comment:
it will be better if you replace these to
```bash
RDEPEND="${PYTHON_DEPS}
    $(python_gen_cond_dep '
        dev-python/awscli[${PYTHON_MULTI_USEDEP}]
        dev-python/boto3[${PYTHON_MULTI_USEDEP}]
        dev-python/coloredlogs[${PYTHON_MULTI_USEDEP}]
        dev-python/pytest-xdist[${PYTHON_MULTI_USEDEP}]
        dev-python/requests[${PYTHON_MULTI_USEDEP}]
    ')"
```

P.S.
recently, any ebuild with `python-single-r1` eclass requires to use:
* `$(python_gen_cond_dep 'foo/bar1[${PYTHON_MULTI_USEDEP}]')` 
or
* `$(python_gen_cond_dep "foo/bar1[\${PYTHON_MULTI_USEDEP}]")` — for multi PYTHON_TARGETS (with double quotes you need escape \\${...} var)

and
 *  `foo/bar1[${PYTHON_SINGLE_USEDEP}]` — for other python packages with `python-single-r1` eclass (like your ebuild) in *DEPEND

See example: 
https://github.com/pentoo/pentoo-overlay/blob/be40ef15ec2ebf1fb3e50f39663176877ef66a51/sys-devel/pwndbg/pwndbg-99999999.ebuild#L32-L50

Reply:
@linxon Could accept the pull request and made these changes yourself? You are the expert in this ;-)